build/AzurePipelinesTemplates/WindowsAppSDK-RunTests-Steps.yml

```diff
@@ -6,27 +6,17 @@ parameters:
   BinaryCompatSwitch: ''
   testLocale: ''
 steps:
-  # Download the BuildOutput from the Build stage(s).
-  # We only bring down the relevant content for this build config (Debug/Release) & platform, to save some space and time.
-  - task: DownloadBuildArtifacts@0
-    displayName: 'Download: BuildOutput'
+  - task: DownloadPipelineArtifact@2
+    displayName: 'Download Foundation x64'
     inputs:
-      artifactName: 'BuildOutput'
-      downloadPath: $(Build.SourcesDirectory)\BuildOutput
+      artifactName: "drop_TransportPackage_BuildBinaries_$(buildConfiguration)_$(buildPlatform)"
+      targetPath: '$(Build.SourcesDirectory)\BuildOutput'
 
-  # We only bring down the relevant content for this build config (Debug/Release) & platform, to save some space and time.
-  - task: DownloadBuildArtifacts@0
-    displayName: 'Download: Test dependencies'
+  - task: CopyFiles@2
+    displayName: MoveToOutputDirectory
```

Comment:
Looks like the source is an "output directory" and the target is not. Is the display name MoveToOutputDirectory fitting?